resources/translations/fi.edn

```diff
@@ -228,6 +231,7 @@
          :actions "Toiminnot"
          :add-comments-not-shown-to-applicant "Kommentoi (ei näy hakijalle)"
          :add-comments-shown-to-applicant "Kommentoi (näkyy hakijalle)"
+         :add-remark "Lisää merkintä (näytetään tai piilotetaan hakijalta)"
```

Comment:
minor: In other Finnish translations, the term used is "huomio" instead of "merkintä".